frontend/src/metabase/visualizations/components/LegendItem.jsx

```diff
@@ -50,10 +51,13 @@ export default class LegendItem extends Component {
         className={cx(
           className,
           "LegendItem",
-          "no-decoration flex align-center fullscreen-normal-text fullscreen-night-text",
+          "fullscreen-normal-text fullscreen-night-text",
+          CS.noDecoration,
```

Comment:
super nit: I'd just preserve the order for ease of comparison and when we're tracking the history.

I'd put `CS.noDecoration` and the other 2 classes above `"fullscreen-normal-text fullscreen-night-text"`